tests/test_data_simulator.py

```diff
@@ -20,6 +20,103 @@ def test_get_schema(default_dictionary):
     # TODO: delete generated files at the end of tests
 
 
+def test_generate_submission_order():
+    datadictionary = DataDictionary(
+        local_file=os.path.join(MOD_DIR, "schemas/gtex.json")
```

Comment:
is this test effectively a regression test for what was previously failing in MIDRC?

Reply:
it's a regression test for BDC for what i broke with my "fix" here https://github.com/uc-cdis/data-simulator/pull/100

Reply:
but it also serves as a generic test for the function, since there were 0 unit tests for this before